### PR TITLE
Question pic display

### PR DESCRIPTION
**Pull request by sapirshp**

### CODE REVIEW

File: app/src/main/java/com/example/palabra/MainActivity.java [hunk 1]
```diff
@@ -25,11 +26,21 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void ask_question()
     {
         final Question current_quest = Question.get_question();
+        switch (current_quest.getCorrectAnsStr())
```
Comment (orenpinto) [inline app/src/main/java/com/example/palabra/MainActivity.java]: Switch case will be problematic when the amount of questions will grow, try to find another way

File: app/src/main/java/com/example/palabra/answerScreen.java [hunk 2]
```diff
@@ -30,8 +32,24 @@ protected void onCreate(Bundle savedInstanceState) {
         is_correct_box.setText(message);
 
         String correct_ans_str = getIntent().getExtras().getString("ANSWER");
-        TextView correct_ans_box = findViewById(R.id.answer_text_view);
-        correct_ans_box.setText(correct_ans_str);
+        //TextView correct_ans_box = findViewById(R.id.answer_text_view);
+        //correct_ans_box.setText(correct_ans_str);
+        Drawable questionImage =  getResources().getDrawable(R.drawable.apple);;
+        switch (correct_ans_str)
```
Comment (orenpinto) [inline app/src/main/java/com/example/palabra/answerScreen.java]: The same thing also in this switch case

File: app/src/main/java/com/example/palabra/MainActivity.java (hunk 1 above)
Comment (sapirshp) [reply app/src/main/java/com/example/palabra/MainActivity.java]: fixed

File: app/src/main/java/com/example/palabra/answerScreen.java (hunk 2 above)
Comment (sapirshp) [reply app/src/main/java/com/example/palabra/answerScreen.java]: fixed

**Pull request merged**